charts/gardener-extension-admission-gcp/values.yaml

```diff
@@ -32,3 +34,8 @@ global:
         -----END RSA PRIVATE KEY-----
   # Kubeconfig to the target cluster. In-cluster configuration will be used if not specified.
   kubeconfig:
+
+  serviceAccountTokenVolumeProjection:
```

Comment:
I think, this deserves some docs for operators to explain, how the feature can be used.
https://github.com/gardener/gardener-extension-provider-gcp/pull/380#issuecomment-1018388887 should already be a good starting point.

You can add the docs to https://github.com/gardener/gardener-extension-provider-gcp/blob/master/docs/usage-as-operator.md, or create a new document that is referenced from there. Currently, there is no explanation about how to actually deploy this extension's components. Maybe this would be a good point to start with that.